pkg/gencred/cmd/main.go

```diff
@@ -0,0 +1,210 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	flag "github.com/spf13/pflag"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/tools/clientcmd/api/latest"
+	"k8s.io/test-infra/pkg/gencred"
+	"sigs.k8s.io/yaml"
+)
+
+const (
+	// defaultContextName is the default context name.
+	defaultContextName = "build"
+	// defaultConfigFileName is the default kubeconfig filename.
+	defaultConfigFileName = "kubeconfig.yaml"
```

Comment:
We should be able to import this from k8s

Reply:
Now *defaulting* to stdout.